backend/src/data/seeds/seed_dev.ts

```diff
@@ -4,7 +4,7 @@ import { AuthService } from "../../services/auth";
 export async function seed(knex: Knex): Promise<void> {
   return knex.transaction(async (trx) => {
     // Deletes ALL existing entries
-    await trx("pools").del();
```

Comment:
Be aware that the pools table still exists, although it is not used currently it will be used in the future